presto-benchto-benchmarks/src/main/resources/benchmarks/presto/tpcds.yaml

```diff
@@ -13,6 +13,8 @@ tpcds_sf1000: tpcds_sf1000_orc
 tpcds_sf3000: tpcds_sf3000_orc
 tpcds_sf10000: tpcds_sf1000_orc
 variables:
+#queries are assigned to groups so they execute within 15-60s each (for an arbitrary benchmark cluster)
```

Comment:
the same comment as for commit message in the other commit. Plase put a space after `#`

Reply:
bump